merp_outgoing_routing/models/res_company.py

```diff
@@ -9,15 +9,24 @@ class Company(models.Model):
 
     outgoing_routing_strategy = fields.Selection(
         [
-            ('name', 'Sort by source locations in alphabetical order'),
-            ('removal_prio', 'Sort by location removal strategy priority'),
-            ('product', 'Sort by product name'),
+            # path should be valid for both stock pickings and quants
+            ('location_id.removal_prio', 'Location removal priority'),
+            ('location_id.name', 'Location name'),
+            ('product_id.name', 'Product name'),
         ],
-        string='Routing Strategy', default='name')
+        string='Routing Strategy', default='location_id.name')
```

Comment:
Have you also checked for compatability With module https://github.com/ventor-tech/merp/blob/13.0/merp_picking_wave_base/models/stock_picking_wave.py#L15

Reply:
it works